swan-lake/learn/observing-ballerina-code.md

```diff
@@ -293,57 +286,53 @@ specification.](https://github.com/opentracing/specification/blob/master/semanti
 
 #### Using the Jaeger Client
 Jaeger is the default tracer supported by Ballerina. Below is the sample configuration options that are available in
 the Jaeger.
 
 ```toml
-[b7a.observability.tracing]
-enabled=true
-provider="jaeger"
-
-[b7a.observability.tracing.jaeger.sampler]
-type="const"
-param=1.0
-
-[b7a.observability.tracing.jaeger.reporter]
-hostname="localhost"
-port=5775
-
-[b7a.observability.tracing.jaeger.reporter.flush.interval]
-ms=2000
-
-[b7a.observability.tracing.jaeger.reporter.max.buffer]
-spans=1000
+[ballerina.observe]
+tracingEnabled=true
+tracingProvider="jaeger"
+
+[ballerinax.jaeger]
+agentHostname="localhost"
+agentPort=6831
+samplerType="const"
+samplerParam=1.0
+reporterFlushInterval=2000
+reporterBufferSize=1000
 ```
 
 The table below provides the descriptions of each configuration option and possible values that can be assigned.
 
 Configuration Key | Description | Default Value | Possible Values 
 --- | --- | --- | --- 
-b7a.observability.tracing. jaeger.reporter.hostname | Hostname of the Jaeger server | localhost | IP or hostname of the Jaeger server. If it is running on the same node as Ballerina, it can be localhost. 
-b7a.observability.tracing. jaeger.reporter.port | Port of the Jaeger server | 5775 | The port to which the Jaeger server is listening.
-b7a.observability.tracing. jaeger.sampler.type | Type of the sampling methods used in the Jaeger tracer. | const | `const`, `probabilistic`, or `ratelimiting`.
-b7a.observability.tracing. jaeger.sampler.param | It is a floating value. Based on the sampler type, the effect of the sampler param varies | 1.0 | For `const` `0` (no sampling) or `1` (sample all spans), for `probabilistic` `0.0` to `1.0`, for `ratelimiting` any positive integer (rate per second).
-b7a.observability.tracing. jaeger.reporter.flush.interval.ms | Jaeger client will be sending the spans to the server at this interval. | 2000 | Any positive integer value.
-b7a.observability.tracing. jaeger.reporter.max.buffer.spans | Queue size of the Jaeger client. | 2000 | Any positive integer value.
+ballerina.observe. agentHostname | Hostname of the Jaeger agent | localhost | IP or hostname of the Jaeger agent. If it is running on the same node as Ballerina, it can be localhost. 
+ballerina.observe. agentPort | Port of the Jaeger agent | 6831 | The port to which the Jaeger agent is listening.
+ballerina.observe. samplerType | Type of the sampling methods used in the Jaeger tracer. | const | `const`, `probabilistic`, or `ratelimiting`.
+ballerina.observe. samplerParam | It is a floating value. Based on the sampler type, the effect of the sampler param varies | 1.0 | For `const` `0` (no sampling) or `1` (sample all spans), for `probabilistic` `0.0` to `1.0`, for `ratelimiting` any positive integer (rate per second).
+ballerina.observe. reporterFlushInterval | Jaeger client will be sending the spans to the server at this interval. | 2000 | Any positive integer value.
```

Comment:
```suggestion
ballerina.observe. reporterFlushInterval | The Jaeger client will be sending the spans to the server at this interval. | 2000 | Any positive integer value.
```

Reply:
Fixed in 146f6434f70aee59d33f47a274e40ed2ac703d6a